hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/insights.tsx

```diff
@@ -155,12 +165,45 @@ export class Insights extends React.Component<Record<string, object>, IInsightsS
         const lowerbound = upperboundPower > 10 ? size(2 ** (upperboundPower - 1)) : size(0);
         return `${lowerbound} - ${upperbound}`;
       });
+      
+      const xyConatainerCountMap: Map<number, number> = containerCountResponse.reduce(
+        (map: Map<number, number>, current) => {
+          const fileSize = current.containerSize;
```

Comment:
```suggestion
          const containerSize = current.containerSize;
```

Reply:
@dombizita 
Completed Change Suggested by you.